front-end/src/pages/OwnerProfile/OwnerProfile.tsx

```diff
@@ -70,7 +71,22 @@ const OwnerProfilePage: React.FC = () =>{
         })
         .catch(err => {
             console.log(err)
-            //setShowFail(true);
+        }) 
+
+        //get number of employees
+        fetch(`https://gym-king.herokuapp.com/owners/employees/${localStorage.getItem("email")}`,{
+            method: 'GET'
+        })
+        .then(response =>response.json())
+        .then(response =>{
+            console.log(response)
+            if(response != null)
```

Comment:
response is empty []

rather jsut say setNumEmployees(response.length)

Reply:
Will do